package.json

```diff
@@ -27,6 +27,7 @@
     "dev": "run-p --print-label dev:*",
     "dev:minimal": "cd examples/minimal && npm run dev",
     "dev:controlpanel": "cd examples/controlpanel && npm run dev",
+    "postinstall": "patch-package",
```

Comment:
Only needed until https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50550#pullrequestreview-566310368 is merged and released